sormas-backend/src/test/java/de/symeda/sormas/backend/person/PersonFacadeEjbTest.java

```diff
@@ -63,16 +68,47 @@
 import de.symeda.sormas.api.user.UserRole;
 import de.symeda.sormas.api.utils.DateHelper;
 import de.symeda.sormas.api.utils.SortProperty;
+import de.symeda.sormas.api.vaccination.VaccinationDto;
 import de.symeda.sormas.backend.AbstractBeanTest;
-import de.symeda.sormas.backend.TestDataCreator.RDCF;
-import de.symeda.sormas.backend.TestDataCreator.RDCFEntities;
+import de.symeda.sormas.backend.MockProducer;
+import de.symeda.sormas.backend.TestDataCreator;
 
 public class PersonFacadeEjbTest extends AbstractBeanTest {
 
+	protected TestDataCreator.RDCF rdcf;
+	protected TestDataCreator.RDCFEntities rdcfEntities;
+	protected UserDto nationalUser;
+
+	/**
+	 * Resets mocks to their initial state so that mock configurations are not
+	 * shared between tests.
+	 */
+
+	@Before
+	@Override
+	public void init() {
```

Comment:
it's not needed to be annotated with @Before because the super method is already annotated and you can call super.init() instead of duplicating it's content here